src/core/app/user.ts

```diff
@@ -1,25 +1,33 @@
 import { appConfig } from '~core/app_config';
 
-export type CurrentUser = {
-  id: string | null;
+export type UserProfileMain = {
   username?: string;
   email?: string;
   firstName?: string;
   lastName?: string;
   fullName?: string;
+  bio?: string;
+};
+
+export type UserProfileSettings = {
   language: string;
   useMetricUnits: boolean;
   subscribedToKonturUpdates: boolean;
-  bio?: string;
   osmEditor: string;
   defaultFeed: string;
   theme: string;
+};
+
+export type UserProfileApi = UserProfileMain & UserProfileSettings;
```

Comment:
```suggestion
export type UserProfileApiResponse = UserProfileMain & UserProfileSettings;
```
?
Or just `UserProfileResponse`

Reply:
to be refactored in next iterations